### DISCUSSION TITLE
Series03. Table03

### DISCUSSION THREAD

**Discussion by cheisbaee** [IBAHCM/RPiR#542] [Registration]
@maja-nagler @RamahKimbu@ @KatarinaCO
@lolliankosyando @cheisbaee

**Comment by RamahKimbu** [comment]
My repository for series03 exercise; https://github.com/SBOHVM/RamahKimbuSeries03.git

**Comment by LollianKosyando** [comment]
My repository for Series 03
https://github.com/SBOHVM/lolliankosyandoSeries03.git

**Comment by maja-nagler** [comment]
Hi :)
This is the link to my third repository: https://github.com/SBOHVM/majanaglerseries3.git

**Reply by RamahKimbu** [reply]
Hi Maja!
Could you please allow access to your repository?

**Reply by maja-nagler** [reply]
Hey :) 
This should be done now (though please note the repository is not finished).

**Reply by RamahKimbu** [reply]
Hi! 
I can now access it, 
Cheers!

**Comment by maja-nagler** [comment]
Hey  @KatarinaCO @cheisbaee, 
could You paste the links to your series 3 repositories?
I cannot find them in the repositories section of SBOHVM.

**Comment by cheisbaee** [comment]
Hi everyone, this is the link to my repository: https://github.com/SBOHVM/cheisbaeeSeries03.git

**Reply by RamahKimbu** [reply]
Hi Che,

When attempting to access your repository, there is a 404 error. Please try to fix the issue.

**Reply by cheisbaee** [reply]
I'm just finishing up 3-6 so it's not complete yet but I've committed everything up to 3-5.

**Reply by RamahKimbu** [reply]
Hi Che,
Perfect! I can now access it.

**Comment by KatarinaCO** [comment]
Hi everyone

here is the link https://github.com/SBOHVM/KatarinaCOSeries03.git
________________________________

**Reply by RamahKimbu** [reply]
Hi Katarina,
There is error 404 on your repository

**Comment by KatarinaCO** [comment]
made it public, try now please?
________________________________

**Comment by KatarinaCO** [comment]
Or try to open it through repositories? https://github.com/orgs/SBOHVM/repositories
________________________________

**Reply by maja-nagler** [reply]
Hey. It is not visible. That is usually an access issue. If You go to your repository, click settings => Collaborators and teams => manage access => add people and add our user names we should be able to access it.
<img width="957" alt="Screenshot 2022-12-14 at 22 10 16" src="https://user-images.githubusercontent.com/118539738/207726313-d922ef60-348a-42c2-b1ed-f5472a78eb97.png">

**Comment by KatarinaCO** [comment]
sorry, did not know!
thank you, it is done 🙂
________________________________
